systems/subscriber/test-agent/pb/test_agent.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+option go_package = "pb/gen";
+
+import "validator.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+
+
+package ukama.test_agent.v1;
```

Comment:
I think we should update packages to ukama.subscriber.test_agent.v1

Reply:
Okay.  Until now, our proto packages paths were set to match the following pattern: `ukama.service_name.version`. I understant that now, we are going to update all of them to match the following pattern: `ukama.system_name.service_name.version`